tests/opensource-suites/dash/base/actionPanel.test.ts

```diff
@@ -0,0 +1,63 @@
+import {Page, expect} from '@playwright/test';
+
+import DashboardPage from '../../../page-objects/dashboard/DashboardPage';
+import {openTestPage, slct} from '../../../utils';
+import {COMMON_SELECTORS} from '../../../utils/constants';
+import datalensTest from '../../../utils/playwright/globalTestDefinition';
+import {WorkbooksUrls} from 'constants/constants';
+import {Workbook} from 'page-objects/workbook/Workbook';
+
+const PARAMS = {
+    DASH_NAME_PREFIX: 'e2e-test-dash',
+    CONTROL_TITLE: 'test-control',
+    CONTROL_FIELD_NAME: 'test-control-field',
+};
+
+datalensTest.describe('Dashboards are Basic functionality', () => {
```

Comment:
Let's correct the title to "Dashboards - Basic functionality" there and below (this is probably a translation lag)